File: Domain/AppTrainer/Tome.cs
using System;

namespace Domain.AppTrainer
{
    public class Tome
    {
        public Guid Id { get; set; }
        public string Title { get; set; }
        public bool Active { get; set; }
        public int Position { get; set; }
        public int TotalEtudes { get; set; }
        public string AppUserId { get; set; }

    }
}

File: client-app/src/app/models/appTrainer/domain/tome.ts
export interface ITome {
  id: string;
  title: string;
  active: boolean;
  position: number;
  totalEtudes: number;
  totalTime: number;
}

export interface ITomeFormValues extends Partial<ITome> {
  // FUTURO USO para elementos opcionais
}

export class TomeFormValues implements ITomeFormValues {
  id?: string = undefined;
  title: string = '';

  constructor(init?: ITomeFormValues) {
    Object.assign(this, init);
  }
}
